client/src/main/scala/io/delta/sharing/client/DeltaSharingClient.scala

```diff
@@ -391,14 +434,15 @@ class DeltaSharingRestClient(
       maxFiles = None,
       pageToken = None,
       includeRefreshToken = None,
-      refreshToken = None
+      refreshToken = None,
+      idempotency_key = None
```

Comment:
nit: use camel case?

Reply:
I found out this issue in the integration testing, because the proto we added was with _ , so this will make it consistant with the proto field.